lib/src/components/NxDropdown/NxDropdown.scss

```diff
@@ -141,6 +145,19 @@
   }
 }
 
+.nx-dropdown-link .nx-icon {
+  &:hover {
+    color: var(--nx-color-link-hover);
+    text-decoration: underline;
+  }
+
+  &.disabled {
```

Comment:
The icon isn't going to have the `disabled` class, so I doubt this block is doing anything

Reply:
It's for the close icon next to the reset button under the filter dropdown. When the filter doesn't have anything selected the reset button and the icon are both disabled.

Reply:
Oh yeah, removed it here: 602eb123d4617a696be4b955b4cdb781fd26ed1b